opencog/benchmark/atomspace_bm.cc

```diff
@@ -25,6 +27,8 @@ int main(int argc, char** argv)
      "          \t(default: 100000)\n"
      "-r <int>  \tLooping count; how many times a python/scheme operation is looped\n"
      "          \t(default: 1)\n"
+     "-R <int>  \tUse specific randomseed; useful for benchmark comparisons\n"
+     "          \t(default: time(NULL))\n"
```

Comment:
The shape of the distribution of numbers to time is really skewed because a very few number to types have many child atoms while others have none. The number of 0s and 1s in the series matters far more than the number of most larger digits.

At least that's my working theory to explain the sensitivity even for 100,000 calls.